README.md

```diff
@@ -129,6 +129,61 @@ env "gorm" {
 }
 ```
 
+### Supported Features
+#### View
+To create a view, you can define the `ViewDef` method on a struct:
+
+```go
+type User struct {
+  gorm.Model
+  Name string
+  Age  int
+}
+
+// this is our view
+type WorkingAgedUsers struct {
+  Name string
+  Age  int
+}
+
+func (WorkingAgedUsers) ViewDef() []gormschema.ViewOption {
+  return []gormschema.ViewOption{
+    gormschema.BuildStmt(func(db *gorm.DB) *gorm.DB {
+      return db.Model(&User{}).Where("age BETWEEN 18 AND 65").Select("name, age")
+    }),
+  }
+}
+```
+Then load the view just like a regular model:
+
+```go
+stmts, err := gormschema.New("mysql").Load(&models.WorkingAgedUsers{}, &models.User{})
+```
+If you need to create a complex view definition, you can utilize the `CreateStmt` option to specify a raw CREATE VIEW query, as shown below:
+```go
+type BotlTracker struct {
+  ID   uint
+  Name string
+}
+
+func (BotlTracker) ViewDef() []gormschema.ViewOption {
+  return []gormschema.ViewOption{
+    gormschema.CreateStmt("CREATE VIEW botl_trackers AS SELECT id, name FROM pets WHERE name LIKE ?", "botl%"),
+  }
+}
+```
+The "view-based" model struct works just like a regular model struct; you can use it in your GORM queries. Just make sure the model's table name is the same as the view's name:
+```go
+var botlFamily []models.BotlTracker
+db.Find(&botlFamily)
+```
+The view's name in the case of using the `BuildStmt` option will adhere to the GORM convention, which is the pluralized struct name (in this case, `working_aged_users`), but it could be customized using the `TableName()` method:"
```

Comment:
```suggestion
The view's name in the case of using the `BuildStmt` option will adhere to the GORM convention, which is the pluralized struct name (in this case, `working_aged_users`), but it could be customized using the `TableName()` method:
```